app/controllers/subscriptions_management_controller.rb

```diff
@@ -0,0 +1,150 @@
+class SubscriptionsManagementController < ApplicationController
+  before_action :set_cache_control_header
+  before_action :check_if_access_is_enabled
+  before_action :get_subscriptions
+
+  MISSING_EMAIL_ERROR = "Please enter your email address.".freeze
+  INVALID_EMAIL_ERROR = "This doesn’t look like a valid email address – check you’ve entered it correctly.".freeze
+  EMAIL_CHANGED_NOTICE = "Your email address has been changed.".freeze
+
+  def index; end
+
+  def update_frequency
+    @back_url = list_subscriptions_path
+    @subscription_id = subscription_id
+    @current_frequency = @subscriptions[@subscription_id]['frequency']
+  end
+
+  def change_frequency
+    # TODO: Enable this
+    # email_alert_api.change_subscription(
+    #   id: subscription_id,
+    #   frequency: new_frequency
+    # )
+    session[:subscriptions][subscription_id]['frequency'] = new_frequency
```

Comment:
I realise this is stubbed out, but just to note that when we change frequency we should probably create a new subscription row in the database and set the current subscription's `ended_at` and `ended_at_reason`.